### DISCUSSION TITLE
How can insert a picture from my computer

### DISCUSSION THREAD

**Discussion by amarangunic** [earthlab-education/Earth-Analytics-AY24#224] [Q&A]
### How can insert a image from my computer to my portofolio? 
I put an image from a webpage with the link of the image, but what happen when the image is in the computer, which is the syntaxis?

Regards

**Comment by eculler** [comment]
Hi @amarangunic -- thanks for the question, it can be tricky to use images from your computer in GitHub Pages.

What you need to do, is click on the `New File` button on GitHub, and instead if starting a new text file like we did to create `index.md` you select `Upload Files`. Then you can browse your computer for the image you want and upload it to GitHub. There's more information about how to do this in [the textbook](https://cu-esiil-edu.github.io/esiil-learning-portal/foundations/pages/03-git-github/03-github-portfolio/02-images.html#upload-an-image-to-github) also.

One thing to be aware of is that if your images are too large you won't be able to upload them. We can talk about alternatives if that comes up for you.

**Reply by BioNCphilic** [reply]
In my experience, the image uploads even if it is not the right size, but it won't display. I had to upload a second file that was smaller to get it to work on my portfolio page.
![image](https://github.com/user-attachments/assets/9cfc3d63-61f5-4962-9482-56e59a603e70)
